src/components/table/table.ts

```diff
@@ -75,7 +75,11 @@ export class MTable extends ModulVue {
     }
 
     public getIconName(columnTable: MColumnTable): string {
-        return !this.sortedColumn || columnTable.dataProp !== this.sortedColumn.dataProp || this.sortedColumn.ascending ? 'm-svg__arrow-thin--up' : 'm-svg__arrow-thin--down';
+        return this.isAscending(columnTable) ? 'm-svg__arrow-thin--up' : 'm-svg__arrow-thin--down';
+    }
+
+    private isAscending(columnTable: MColumnTable): boolean {
```

Comment:
C'est étrange, tes 2 conditions n'étaient pas identiques auparavant. 
Tu avais: 
`!this.sortedColumn || columnTable.dataProp !== this.sortedColumn.dataProp || **!this.sortedColumn.ascending**`
vs
`!this.sortedColumn || columnTable.dataProp !== this.sortedColumn.dataProp || **this.sortedColumn.ascending**`

Mais maintenant, tu les utilises comme si elles étaient pareil :O

Reply:
En effet, je suis allé trop vite. C'est "reverté"